homework-submission/react-hyf/server/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "\"express, nodemon and concurrently for react-hyf\"",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
```

Comment:
It's a very nice thing to see a complete application, since you still have the server side, you have two packages aren't in use `concurrently` and `nodemon`.